axelor-crm/src/main/resources/domains/AppCrm.xml

```diff
@@ -5,6 +5,8 @@
 	<module name="base" package="com.axelor.apps.base.db"/>
 
 	<entity name="AppCrm" lang="java" extends="App">
+	<integer name="assignableUser" title="Assignable User" selection="crm.app.crm.assignable.user.select"/>
```

Comment:
title is 'Assignable users' not 'Assignable User', change it and field name too accordingly